www-servers/nginx-unit/metadata.xml

```diff
@@ -16,6 +16,12 @@
 		reconfiguration of specific application parameters as needed by the
 		engineering or operations.
 	</longdescription>
+	<use>
+		<flag name="php5-6">Support for PHP 5.6</flag>
```

Comment:
Why are you using custom flags instead of reusing PHP_TARGETS?

Reply:
See https://bugs.gentoo.org/665362#c13

Reply:
This whole thing looks fishy to me. But *shrug*.